drake/examples/SimpleCar/test/CMakeLists.txt

```diff
@@ -2,4 +2,4 @@ if (GTEST_FOUND AND LCM_FOUND AND NOT WIN32)
   add_executable(simple_car_test simple_car_test.cc)
   target_link_libraries(simple_car_test ${GTEST_BOTH_LIBRARIES} drakeSimpleCar)
   add_test(NAME simple_car_test COMMAND simple_car_test)
-endif (GTEST_FOUND AND NOT WIN32)
+endif (GTEST_FOUND AND LCM_FOUND AND NOT WIN32)
```

Comment:
This is fine, but you could also go crazy and change this to just
```
endif()
```
and avoid duplication of the condition altogether!